interface/CH.json

```diff
@@ -4169,7 +4169,70 @@
         }
       }
     },
-    "/ClientCertificates/{certificate}": {
+    "/ClientCertificates/{id}": {
+      "get": {
+        "summary": "Get a specific client certificate",
+        "description": "Get a specific `ClientCertificate` object.",
+        "tags": [
+          "ClientCertificate"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "The id of the client certificate to get. It must be the certificate's case-insensitive thumbprint and not contains space character.",
```

Comment:
The id of the client certificate to get. The Id must be the certificate's thumbprint without spaces, dashes, or any other special characters. The thumbprint is case-insensitive.